CMakeLists.txt

```diff
@@ -79,6 +79,11 @@ include(external/bootstrap.cmake)
  
 # Now build nupic_core project.
 set(core_library   nupic_core)
+if(MSVC)
+    set(src_lib_shared)
```

Comment:
comment that this disables shared for Win

Reply:
ok